Tests/MessagingPush/APITest.swift

```diff
@@ -0,0 +1,28 @@
+@testable import CioMessagingPush
+@testable import CioTracking
+import Foundation
+import SharedTests
+import XCTest
+
+/**
+ Contains an example of every public facing SDK function call. This file helps
+ us prevent introducing breaking changes to the SDK by accident. If a public function
+ of the SDK is modified, this test class will not successfully compile. By not compiling,
+ that is a reminder to either fix the comilation and introduce the breaking change or
```

Comment:
This comment makes sense to me. On line 11, do you mean "...compilation and introduce..." ?